Felienne/hedy
# Adding 'breakpoints' in editor

**Description**
For my bachelor project #1329, I am trying to give kids more tools to debug their programs. A powerful tool is a breakpoint. With this PR, users can click on the line gutter to create a breakpoint. A sleep emoji such as "😴" will appear on the specific line gutter, and the code will be struck through. 

The code that has a breakpoint will **not** be executed! Hedy will ignore all the lines with a sleeping emoji. 

**Screenshots**  

https://user-images.githubusercontent.com/33595265/162415316-1f86baf2-3893-4f54-8fd7-49de7a5c1e7c.mov


It can work with any kind of command/keyword! 

https://user-images.githubusercontent.com/33595265/162414920-d425e56b-a7e1-447c-ac90-24e187b6c10d.mov




**How to test**
Go to any level, and try to click on a number in the line gutter in the editor. A sleep emoji should appear. Try to run the code. Hedy should ignore the line you have selected! (If its not working, you might need to recompile the Typescript code and clear the cache in your browser)

Extra tests:

- Make sure you are able to create and delete breakpoints, and that Hedy reacts accordingly when running the code
- Create some breakpoints and change the adventure. Check whether the breakpoints have disappeared
